demo/apiExamples.md

```diff
@@ -1,169 +1,190 @@
+<auro-button onClick="showToast('#defaultToast')">Show default toast</auro-button>
+<auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast"> Default toast with no error type  </auro-toast>
+<auro-button onClick="showToast('#defaultToast-noIcon')">Show default toast with no icon</auro-button>
+<auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast-noIcon" noIcon> Default toast with no error type  </auro-toast>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Property Examples
+
+#### <a name="propertyName"></a>`propertyName`<a href="#auro-toast" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>
 Explanation and use description.
 
 <div class="exampleWrapper">
   <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
   <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
-  <auro-button onClick="toggleToast()">Show default toast</auro-button>
-  <auro-toast>Toast content goes here.</auro-toast>
+  <auro-button onClick="showToast('#defaultToast')">Show default toast</auro-button>
+  <auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast"> Default toast with no error type  </auro-toast>
+  <auro-button onClick="showToast('#defaultToast-noIcon')">Show default toast with no icon</auro-button>
+  <auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast-noIcon" noIcon> Default toast with no error type  </auro-toast>
   <!-- AURO-GENERATED-CONTENT:END -->
 </div>
 <auro-accordion lowProfile justifyRight>
-  <span slot="trigger">See code</span>
+  <span slot="trigger">See code</span>
 <!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
-<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
-
+<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
+
 ```html
-<auro-button onClick="toggleToast()">Show default toast</auro-button>
-<auro-toast>Toast content goes here.</auro-toast>
-```
-<!-- AURO-GENERATED-CONTENT:END -->
-</auro-accordion>
-
-### Method Examples
-
-#### <a name="methodName"></a>`methodName`<a href="#auro-toast" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>
+<auro-button onClick="showToast('#defaultToast')">Show default toast</auro-button>
+<auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast"> Default toast with no error type  </auro-toast>
+<auro-button onClick="showToast('#defaultToast-noIcon')">Show default toast with no icon</auro-button>
+<auro-toast style="display: block; margin: 0.5rem 0;"  id="defaultToast-noIcon" noIcon> Default toast with no error type  </auro-toast>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Method Examples
+
+#### <a name="methodName"></a>`methodName`<a href="#auro-toast" style="float: right; font-size: 1rem; font-weight: 100;">back to top</a>

```

Comment:
Please review this document, I am not sure this is doing what we may expect it to do here.